ui/src/app/applications/components/application-conditions/application-conditions.tsx

```diff
@@ -15,7 +15,8 @@ export const ApplicationConditions = ({conditions}: {conditions: models.Applicat
                         <div className={`argo-table-list__row application-conditions__condition application-conditions__condition--${getConditionCategory(condition)}`} key={index}>
                             <div className='row'>
                                 <div className='columns small-2'>{condition.type}</div>
-                                <div className='columns small-10'>{condition.message}</div>
+                                <div className='columns small-8'>{condition.message}</div>
+                                <div className='columns small-2'>{condition.lastTransitionTime}</div>
```

Comment:
Please use `<Timestamp date={condition.lastTransitionTime}/>` to render date in the user time zone. Example: https://github.com/argoproj/argo-cd/blob/a0f39034182e4a82f53bb5ce684e4e0d45bcb1c8/ui/src/app/applications/components/application-deployment-history/application-deployment-history.tsx#L37